src/modules/dashboard/components/UnlockTokenDialog/UnlockTokenForm.tsx

```diff
@@ -39,6 +39,14 @@ const MSG = defineMessages({
     defaultMessage: `You do not have the {roleRequired} permission required
       to take this action.`,
   },
+  unlockedTitle: {
+    id: 'dashboard.UnlockTokenDialog.UnlockTokenForm.dialogunlockedTitle',
```

Comment:
Typo here, should be `.unlockedTitle`

Reply:
Good one, thanks for spotting, fixed on new commit